sdk-api-src/content/d3d12shader/ns-d3d12shader-d3d12_shader_input_bind_desc.md

```diff
@@ -88,6 +88,7 @@ A <a href="/windows/desktop/api/d3dcommon/ne-d3dcommon-d3d_srv_dimension">D3D_SR
 
 The number of samples for a multisampled texture; when a texture isn't multisampled, the value is set to -1 (0xFFFFFFFF).
             This is zero if the shader resource is not a recognized texture.
+            If the resource is a structured buffer the field returns the stride of the struct.
```

Comment:
"Returns" doesn't sound like the correct term for a structure field, as it's the terminology used for functions (which return flow control as well as optionally a value to the caller). The two lines above reference the field as a container ("the value is set") or as a value directly ("this is zero"), I think "is set to" is applicable here too.